packages/api-elasticsearch-tasks/src/settings/EnableIndexing.ts

```diff
@@ -11,10 +11,11 @@ export class EnableIndexing {
 
     public async exec(index: string, settings: IIndexSettingsValues): Promise<void> {
         try {
+            const refreshInterval = parseInt(settings.refreshInterval || "", 10) || 0;
             await this.settings.setSettings(index, {
                 ...settings,
                 numberOfReplicas: settings.numberOfReplicas < 1 ? 1 : settings.numberOfReplicas,
```

Comment:
Just a reminder that, for the 5.39 data migration, we actually ignored the `numberOfReplicas` param when disabling/enabling indexing it all worked just fine:

https://github.com/webiny/webiny-js/pull/4187/files#diff-69c88efb045520b15afd8acdcb24c33eb0877e333c2592a5d10f2981047eae76